i18n/en.json

```diff
@@ -162,28 +162,17 @@
   "add_user_to_channel_modal.title": "Add {name} to a channel",
   "add_users_to_team.title": "Add New Members To {teamName} Team",
   "admin.advance.cluster": "High Availability",
-  "admin.advance.experimental": "Experimental",
```

Comment:
In this files, looks like there are 31 strings that has been removed from the original admin console. That means that we have less configuration than before. Maybe some of them is because the reordering, but 31 means that probably there are sections entirely removed.

Reply:
I also find this unsettling, but I can't point to the exact reasons for the reduction in strings until unless I audit the entire file. I was manually comparing throughout development, but I believe your string extraction utility may have identified some outdated strings - I could be wrong. I will look this over manually once more.

Reply:
Any place where we've combined 2 pages into a single page would remove the page title strings as well.

Reply:
Ok, then this difference makes sense.